assets/src/block-templates/homepage/template.js

```diff
@@ -0,0 +1,33 @@
+import gravityFormWithText from '../templates/gravity-form-with-text';
+
+const {__} = wp.i18n;
+
+const template = () => (
+  [
+    ['core/group', {}, [
+      ['planet4-blocks/carousel-header'],
+      ['planet4-block-templates/issues', {
+        title: __('The issues we work on', 'planet4-blocks'),
```

Comment:
Are we using translation functions for these strings?

Reply:
Yes, we use it in some cases.